client/js/screens/SignUp/styles.js

```diff
@@ -1,9 +1,44 @@
-import { StyleSheet } from "react-native";
-import { globalStyles } from "../../../config/styles";
+import { StyleSheet, Dimensions } from "react-native";
+import { globalStyles } from "../../config/styles";
+var width = Dimensions.get("window").width;
+var height = Dimensions.get("window").height;
 
 const styles = StyleSheet.create({
-  container: globalStyles.container
+  container: globalStyles.container,
+  register: {
+    fontSize: 25,
+    fontWeight: "bold",
+    color: "white",
+    marginTop: 30,
+    textAlign: "center",
+    marginBottom: 30
```

Comment:
use flexbox not marginTop and marginBottom with value 30.

Reply:
i can't see my page cause i'll be needing vanessa's pr "apollo provider" i'll change it once i can navigate to my screen.

Reply:
Then how did you work on it?